build/parent-pom/pom.xml

```diff
@@ -21,13 +21,14 @@
     <parent>
         <groupId>za.co.absa.spline</groupId>
         <artifactId>package-pom</artifactId>
```

Comment:
Shouldn't the name be `spline-root-pom`?

Reply:
As it might be promoted to the ABSA OSS level and hence eventually evicted from the Spline group of artifacts, I though it would be better to keep the artifact name unchanged to not pollute the space with another short living name.